### ISSUE TITLE
Use multiple XSD files

### ISSUE THREAD

**Issue by omaralqady** [amouat/xsd-validator#15]
This doesn't seem to be possible with the current implementation, although it would be quite useful to be able to validate XML against multiple XSD files for more complex applications.

**Comment by amouat** [comment]
True. I don't actively develop this anymore, but I'll leave this open in case anyone wants to take it on.
